e2e/test_claim_token_api.js

```diff
@@ -0,0 +1,404 @@
+import http from 'k6/http';
+import { check, group, sleep } from 'k6';
+import { Rate, Counter } from 'k6/metrics';
+
+const BASE_URL = __ENV.BASE_URL || 'http://127.0.0.1:8787';
+const API_ENDPOINT = `${BASE_URL}/api/claim_token`;
+const CALIBNET_RPC =
+  __ENV.CALIBNET_RPC ||
+  __ENV.FAUCET_TX_URL_CALIBNET ||
+  'https://api.calibration.node.glif.io/rpc/v1';
+
+const TEST_ADDRESSES = {
+  FIL_FORMAT_ADDRESS: 't1pxxbe7he3c6vcw5as3gfvq33kprpmlufgtjgfdq',
+  ETH_FORMAT_ADDRESS: '0xAe9C4b9508c929966ef37209b336E5796D632CDc',
+  INVALID_ADDRESS: 'invalid_address_123',
+  MAINNET_ADDRESS: 'f1mwllxrw7frn2lwhf4u26y4f3m7f6wsl4i3o3jvi',
+};
+
+export const options = {
+  vus: 1,
+  iterations: 1,
+  duration: '6m',
+  thresholds: {
+    'checks': ['rate==1.0'],
+    'test_run_success_rate': ['rate==1.0'],
+  },
+};
+
+const testRunSuccessRate = new Rate('test_run_success_rate');
+const responseTracker = new Counter('api_responses');
+
+const API_TESTS = [
+  {
+    name: 'Invalid Faucet Type',
+    data: {
+      faucet_info: "InvalidFaucet",
+      address: TEST_ADDRESSES.FIL_FORMAT_ADDRESS
+    },
+    expectPattern: 'unknown variant',
```

Comment:
`expectPattern` is not an appropriate name here, `expectedMsg` would be better

Reply:
I added this because, I was not planning to match the entire message just some part (Important parts) of the message hence `expectPattern` but will change it